lib/scss-preprocessor/broccoli-import-theme-components.js

```diff
@@ -11,6 +11,9 @@ function Import(inputTree, options) {
 
   this.include = options.include;
   this.main = options.main;
+  this.themeName = options.themeName || 'ui-base';
+  this.themePrefix = options.themePrefix || 'base';
```

Comment:
I'd like to remove this in the future by actually using a prefix for the base theme too, so it's not special.